actors/eam/src/lib.rs

```diff
@@ -104,6 +112,12 @@ pub struct Create2Params {
     pub salt: [u8; 32],
 }
 
+#[derive(Serialize_tuple, Deserialize_tuple)]
+pub struct Create3Params {
+    #[serde(with = "strict_bytes")]
+    pub initcode: Vec<u8>,
```

Comment:
I'd drop the wrapping struct (as we do with invoke). That way, _all_ messages from the EthAccount actor will have the same parameter structure (cbor-encoded byte array) which will simplify signature verification.